README.md

```diff
@@ -0,0 +1,33 @@
+# SEDaily Miner
+
+A script to download posts and auxilliary post details from software engineering daily via the Wordpress api: https://softwareengineeringdaily.com/wp-json/wp/v2/posts and store it into the DB, to be accessed by the sedaily API.
+
+# Set up
+ - npm install
+ - cp .env.example .env
+
+**Note**: Tested on Node version 7.10.1
+
+ #### - Storing all the Posts first:
+ - `node index.js`
+
+ #### - Adding Image links for Posts:
+ - `node getImg.js`
+
+ #### - Getting Mp3 file links for Posts:
+ - `node getMp3.js`
+
+ #### - Fetching list of all tags:
```

Comment:
@jasonify  For the record, all these files work. I used them to index my local sedaily mongodb instance